### ISSUE TITLE
`checker-emailer` command options and documentation are a mess

### ISSUE THREAD

**Issue by dpk** [DSpace/DSpace#10008]
<https://wiki.lyrasis.org/display/DSDOC7x/Validating+CheckSums+of+Bitstreams#ValidatingCheckSumsofBitstreams-AutomatedChecksumCheckers'Results>

1. The long options are all named with capital letters at the start, which is not usual command-line tool style (it doesn’t match the style used by other `dspace` sub-commands. This would be okay – just a little weird – except also …
2. The `--Not Processed` option has a space in the name. This means you have to enter it at the shell at `'--Not Processed'` or `--Not\ Processed`, which is far from intuitive (how many users of DSpace are not command line experts and don’t even know how to insert spaces within a single token?)
3. There’s an option called `--Unchanged` which says its job is to ‘Send the Unchecked bitstream report’. Why is it called Un*changed* if it actually has to do with Un*checked* things? Which is correct, the option name or the documentation?

**Comment by tdonohue** [comment]
Agreed, it'd be nice to find a volunteer to update/modernize these.  I believe this entire class is mostly from 10+ years ago (which is why these are messy), and it hasn't been maintained in a very long time.

Needs a volunteer to investigate & create a PR to fix the options.

**Comment by minurmin** [comment]
Perhaps this issue could be expanded to include [documentation](https://wiki.lyrasis.org/display/DSDOC7x/Validating+CheckSums+of+Bitstreams) and output regarding checker as well? E.g. on follows: 

 - Clarify which options can be used in combination with others (i.e. verbose or prune) and which are mutually exclusive (apparently rest of the modes). In particular: with looping modes `-l` or `-L` , additional parameters like `-c` or `-d` (implying limited run time or count) are silently accepted but have no effect. An error message (ambiguous mode etc.) or at least a warning that script runs through all items would be more intuitive.
 - With checker `-a` parameter is used to indicate a handle, even though with many other command line utils (index-discovery, filter-media, metadata-export) `-i` is used instead.

**Comment by minurmin** [comment]
This is hilarious. The longer option for `-u` has been spelled in the code as  `--Unchecked` as long as the DSpace commit history goes - at least since [2006](https://github.com/DSpace/DSpace/blob/27d72438fe041acf3aa57f18c1055686ec9e8ca1/dspace/src/org/dspace/checker/DailyReportEmailer.java#L188). So this detail was just a glitch in documentation (corrected to 7x-8x [wiki](https://wiki.lyrasis.org/display/DSDOC8x/Validating+CheckSums+of+Bitstreams) ). 
However, other issues with options (capitalization, space in `Not Processed`) still remain.
